keras/src/ops/image.py

```diff
@@ -76,6 +76,147 @@ def rgb_to_grayscale(images, data_format=None):
+            It can be either `"channels_last"` or `"channels_first"`.
+            `"channels_last"` corresponds to inputs with shape
+            `(batch, height, width, channels)`, while `"channels_first"`
+            corresponds to inputs with shape `(batch, channels, height, width)`.
+            If not specified, the value will be interpreted by
+            `keras.config.image_data_format`. Defaults to `None`.
+
+    Returns:
+        HSV image or batch of HSV images.
+
+    Examples:
+
+    >>> import numpy as np
+    >>> from keras.src import ops
+    >>> x = np.random.random((2, 4, 4, 3))
+    >>> y = ops.image.rgb_to_hsv(x)
+    >>> y.shape
+    (2, 4, 4, 3)
+
+    >>> x = np.random.random((4, 4, 3)) # Single RGB image
+    >>> y = ops.image.rgb_to_hsv(x)
+    >>> y.shape
+    (4, 4, 3)
+
+    >>> x = np.random.random((2, 3, 4, 4))
+    >>> y = ops.image.rgb_to_hsv(x, data_format="channels_first")
+    >>> y.shape
+    (2, 3, 4, 4)
+    """
+    if any_symbolic_tensors((images,)):
+        return RGBToHSV(data_format=data_format).symbolic_call(images)
+    return backend.image.rgb_to_hsv(images, data_format=data_format)
+
+
+class HSVToRGB(Operation):
+    def __init__(self, data_format=None):
+        super().__init__()
+        self.data_format = backend.standardize_data_format(data_format)
+
+    def call(self, images):
+        return backend.image.hsv_to_rgb(images, data_format=self.data_format)
+
+    def compute_output_spec(self, images):
+        images_shape = list(images.shape)
+        dtype = images.dtype
+        if len(images_shape) not in (3, 4):
+            raise ValueError(
+                "Invalid images rank: expected rank 3 (single image) "
+                "or rank 4 (batch of images). "
+                f"Received: images.shape={images_shape}"
+            )
+        if not backend.is_float_dtype(dtype):
+            raise ValueError(
+                "Invalid images dtype: expected float dtype. "
+                f"Received: images.dtype={dtype}"
+            )
+        return KerasTensor(shape=images_shape, dtype=images.dtype)
+
+
+@keras_export("keras.ops.image.hsv_to_rgb")
+def hsv_to_rgb(images, data_format=None):
+    """Convert HSV images to RGB.
+
+    `images` must be of float dtype, and the output is only well defined if the
+    values in `images` are in `[0, 1]`.
+
+    Args:
+        images: Input image or batch of images. Must be 3D or 4D.
+        data_format: A string specifying the data format of the input tensor.
+            It can be either `"channels_last"` or `"channels_first"`.
+            `"channels_last"` corresponds to inputs with shape
+            `(batch, height, width, channels)`, while `"channels_first"`
+            corresponds to inputs with shape `(batch, channels, height, width)`.
+            If not specified, the value will be interpreted by
+            `keras.config.image_data_format`. Defaults to `None`.
```

Comment:
"the value will default to `keras.config.image_data_format()`"

Reply:
Fixed and applied to all other image ops.